terraform/iam.tf

```diff
@@ -83,13 +83,7 @@ data "aws_iam_policy_document" "sns_policy_doc" {
       "sns:Publish",
       "sns:GetTopicAttributes"
     ]
-    resources = [
-      aws_sns_topic.unhandled_events.arn,
-      aws_sns_topic.suspensions.arn,
-      aws_sns_topic.dlq.arn,
-      aws_sns_topic.nems_audit.arn,
-      aws_sns_topic.re_registrations_topic.arn,
-    ]
+    resources = [local.sns_topic_arns]
```

Comment:
Does this need to be surrounded by square brackets?

Reply:
No it does not! Good spot :)